contracts/KlerosLiquidProxy.sol

```diff
@@ -0,0 +1,245 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./interfaces/IKlerosLiquidProxy.sol";
+import "./interfaces/IExtendedKlerosArbitrator.sol";
+
+contract KlerosLiquidProxy is IKlerosLiquidProxy {
```

Comment:
I think we want KlerosLiquidProxy to inherit Multicall? (please excuse my adhoc comments)

Reply:
Yep

Reply:
https://github.com/api3dao/claims-manager/pull/27/commits/45088ce138fc20ce1f79c0ba0e454be8eef29fd9